daemon/src/peer/sync.rs

```diff
@@ -26,18 +26,18 @@ where
         .collect::<Vec<_>>();
 
     for urn in urns {
-        log::debug!("Starting fetch of {} from {}", urn, remote_peer);
+        tracing::debug!(?urn, ?remote_peer, "startin fetch");
```

Comment:
```suggestion
        tracing::debug!(?urn, ?remote_peer, "starting fetch");
```

Alternatively, `"startin'"` :stuck_out_tongue_winking_eye: